src/pages/Items.jsx

```diff
@@ -17,6 +17,12 @@ const getItems = ({ page, pageSize, orderBy, keyword }) => {
   ).then((response) => response.json());
 };
 
+export function displayDefault(isMobile, isTablet) {
+  if (isMobile) return 4;
+  if (isTablet) return 6;
+  return 10;
```

Comment:
```suggestion
const MOBILE_SIZE = 4;
const TABLE_SIZE = 6;
const DESKTOP_SIZE = 10;

  if (isMobile) return MOBILE_SIZE;
  if (isTablet) return TABLE_SIZE;
  return DESKTOP_SIZE;
```

4,6,10같은 정수는 의미가 불분명해서
변수에 담으면 의미를 명확하게 할 수 있습니다